src/ios/SQLitePlugin.m

```diff
@@ -195,6 +208,14 @@ -(void)openNow: (CDVInvokedUrlCommand*)command
                 } else {
                     NSLog(@"ERROR reading sqlite master table");
                     pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:@"Unable to open DB with key"];
+                    
+                    if(selfHealingEnabled) {
+                        [[command.arguments objectAtIndex:0] setObject:dbfilename forKey:@"path"];
+                        [_logger logError:[NSString stringWithFormat:@"iOS database will be deleted to self heal"] withModule:@"SQLite"];
+                        [self deleteNow:command];
+                        return [self openNow:command];
+                    }
+                    
```

Comment:
We had a lot of error messages in Android, don't we have them in iOS too? Or are we not logging it for some reason?

Reply:
Good catch, I will fix this 😄

Reply:
The Android and iOS implementation are very different from each other, I think that we should tackle this in a new issue just for the logging! What do you think?

Reply:
If we don't do this now, we will deliver an half-solution and we might never tackle this in short-term. The logs weren't in the scope of this issue neither and they were added into the Android solution. It doesn't seem right to me. Also, we are talking about a couple of log messages, isn't that so hard to do that we can't do it now?